exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -81,6 +81,10 @@ type AzureManagedControlPlaneSpec struct {
 	// AadProfile is Azure Active Directory configuration to integrate with AKS for aad authentication.
 	// +optional
 	AADProfile *AADProfile `json:"aadProfile,omitempty"`
+
+	// LoadBalancerProfile is the profile of the cluster load balancer.
+	// +optional
+	LoadBalancerProfile *LoadBalancerProfile `json:"loadBalancerProfile,omitempty"`
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1640#discussion_r706664686

Reply:
Reverted change to v1a3 according to your suggestion :)